client/src/research/Transcribe.js

```diff
@@ -0,0 +1,20 @@
+function requestTranscript(token,audioID) {
+  return fetch('/server/research/transcribe/'+audioID+'.wav', {
+    headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+      'x-teachermoments-token': token,
+    },
+    method: 'POST'
+  })
+    .then(result => {
+      return result.json();
+    })
+    .catch(err => {
+      console.log('Failed to request transcript');
```

Comment:
should this return a failure status for the HTTP request?

Reply:
hmmmm. So I think I generally have the return statuses be set on the server side. In this case, in getAudio.js, the insecureStreamAudioFileFromS3 returns a 200 by default if it can find the audio file and returns a 503 if it can't get an audio file from s3.